Live/LiveEventWithDVR/Program.cs

```diff
@@ -1,663 +1,562 @@
+                           StorageAccountName, StorageAccountKey);
 
-            try
-            {
-                await RunAsync(config);
-            }
-            catch (Exception exception)
-            {
-                if (exception.Source.Contains("ActiveDirectory"))
-                {
-                    Console.Error.WriteLine("TIP: Make sure that you have filled out the appsettings.json or .env file before running this sample.");
-                }
-
-                Console.Error.WriteLine($"{exception.Message}");
-
-                if (exception.GetBaseException() is ErrorResponseException apiException)
-                {
-                    Console.Error.WriteLine(
-                        $"ERROR: API call failed with error code '{apiException.Body.Error.Code}' and message '{apiException.Body.Error.Message}'.");
-                }
-            }
 
-            Console.WriteLine("Press Enter to exit the sample.");
-            Console.ReadLine();
 
+        storageClient = new BlobContainerClient(storageConnectionString, blobContainerName);
+        processorClient = new EventProcessorClient(storageClient, consumerGroup, eventHubsConnectionString, eventHubName);
+        mediaEventProcessor = new MediaServicesEventProcessor(null, null, liveEventName);
+
+        processorClient.ProcessEventAsync += mediaEventProcessor.ProcessEventsAsync;
+        processorClient.ProcessErrorAsync += mediaEventProcessor.ProcessErrorAsync;
+
+        await processorClient.StartProcessingAsync();
+    }
+    catch (Exception e)
+    {
+        Console.WriteLine("Failed to connect to Event Hub, please refer README for Event Hub and storage settings. Skipping event monitoring...");
+        Console.WriteLine(e.Message);
+    }
+
+    Console.WriteLine("Creating the LiveEvent, please be patient as this can take time to complete async.");
+    Console.WriteLine("Live Event creation is an async operation in Azure and timing can depend on resources available.");
+
+    // When autostart is set to true, the Live Event will be started after creation. 
+    // That means, the billing starts as soon as the Live Event starts running. 
+    // You must explicitly call Stop on the Live Event resource to halt further billing.
+    // The following operation can sometimes take awhile. Be patient.
+    // On optional workflow is to first call allocate() instead of create. 
+    // https://docs.microsoft.com/en-us/rest/api/media/liveevents/allocate 
+    // This allows you to allocate the resources and place the live event into a "Standby" mode until 
+    // you are ready to transition to "Running". This is useful when you want to pool resources in a warm "Standby" state at a reduced cost.
+    // The transition from Standby to "Running" is much faster than cold creation to "Running" using the autostart property.
+    // Returns a long running operation polling object that can be used to poll until completion.
```

Comment:
Is this comment still valid? How does the long running polling interval now work from the headers?